warehouse/postgres/postgresQueryBuilder.go

```diff
@@ -0,0 +1,62 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+)
+
+type QueryExecution struct {
```

Comment:
Should we rename it to `QueryParams` or maybe `Params` ?

Reply:
QueryParams suites good.